packages/contracts/contracts/BooleanIdentityDecider.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* External Imports */
+import "openzeppelin-solidity/contracts/math/Math.sol";
+
+/* Internal Imports */
+import {DataTypes as types} from "./DataTypes.sol";
+import {UniversalAdjudicator} from "./UniversalAdjudicator.sol";
+
+/**
+ * @title BooleanIdentityDecider
+ * @notice TODO
+ */
+contract BooleanIdentityDecider {
+    struct BooleanIdentityInput {
+        bool inputBool;
+    }
+
+    UniversalAdjudicator adjudicator;
+
+    constructor(address _adjudicator) public {
+        adjudicator = UniversalAdjudicator(_adjudicator);
+    }
+
+    function decideTrue(BooleanIdentityInput memory _input, bytes memory _witness) public {
+        // Check that the input is `true`
```

Comment:
Looks good! Small suggestion is that I think this can be DRY'd (Can pull the common logic between `decideTrue` and `decideFalse` out into a private function that takes in true/false and `require(...)`s and decides via `adjudicator.decideProperty(...)` according to the input boolean).